qa/fhir-server/configmaps.yml

```diff
@@ -6,7 +6,7 @@ metadata:
 data:
   SPRING_DATASOURCE_URL: jdbc:postgresql://postgres-server-qa:5432/clin_fhir_qa?sslmode=verify-full&sslrootcert=/opt/ca/ca.pem
   BIO_AUTH_SERVER_URL: https://auth.qa.clin.ferlab.bio/auth
-  BIO_ELASTICSEARCH_PATIENT_INDEX: prescriptions
```

Comment:
not used, wrong name, default values in API conf are:
BIO_ELASTICSEARCH_PRESCRIPTIONS_INDEX = prescriptions
BIO_ELASTICSEARCH_PATIENTS_INDEX = patients